NPE when forming jgroups cluster
I just got this NPE when deploying a new application on my Wildfly 10 CR4 domain cluster.
JGRP000027: failed passing message up: java.lang.RuntimeException: java.lang.NullPointerException
        at org.jgroups.blocks.MessageDispatcher$ProtocolAdapter.up(MessageDispatcher.java:689)
        at org.jgroups.JChannel.up(JChannel.java:738)
        at org.jgroups.stack.ProtocolStack.up(ProtocolStack.java:1029)
        at org.jgroups.protocols.FORK.up(FORK.java:133)
        at org.jgroups.protocols.FRAG2.up(FRAG2.java:165)
        at org.jgroups.protocols.FlowControl.up(FlowControl.java:394)
        at org.jgroups.protocols.pbcast.GMS.installView(GMS.java:735)
        at org.jgroups.protocols.pbcast.ParticipantGmsImpl.handleViewChange(ParticipantGmsImpl.java:140)
        at org.jgroups.protocols.pbcast.GMS.up(GMS.java:925)
        at org.jgroups.stack.Protocol.up(Protocol.java:412)
        at org.jgroups.protocols.pbcast.STABLE.up(STABLE.java:294)
        at org.jgroups.protocols.UNICAST3.up(UNICAST3.java:474)
        at org.jgroups.protocols.pbcast.NAKACK2.deliverBatch(NAKACK2.java:982)
        at org.jgroups.protocols.pbcast.NAKACK2.removeAndPassUp(NAKACK2.java:912)
        at org.jgroups.protocols.pbcast.NAKACK2.handleMessage(NAKACK2.java:846)
        at org.jgroups.protocols.pbcast.NAKACK2.up(NAKACK2.java:618)
        at org.jgroups.protocols.VERIFY_SUSPECT.up(VERIFY_SUSPECT.java:155)
        at org.jgroups.protocols.FD.up(FD.java:260)
        at org.jgroups.protocols.FD_SOCK.up(FD_SOCK.java:310)
        at org.jgroups.protocols.MERGE3.up(MERGE3.java:285)
        at org.jgroups.protocols.Discovery.up(Discovery.java:295)
        at org.jgroups.protocols.TP.passMessageUp(TP.java:1577)
        at org.jgroups.protocols.TP$MyHandler.run(TP.java:1796)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NullPointerException
        at org.wildfly.clustering.server.group.ChannelNodeFactory.createNode(ChannelNodeFactory.java:55)
        at org.wildfly.clustering.server.group.ChannelNodeFactory.createNode(ChannelNodeFactory.java:40)
        at org.wildfly.clustering.server.dispatcher.ChannelCommandDispatcherFactory.getNodes(ChannelCommandDispatcherFactory.java:207)
        at org.wildfly.clustering.server.dispatcher.ChannelCommandDispatcherFactory.getNodes(ChannelCommandDispatcherFactory.java:201)
        at org.wildfly.clustering.server.dispatcher.ChannelCommandDispatcherFactory.viewAccepted(ChannelCommandDispatcherFactory.java:216)
        at org.jgroups.blocks.MessageDispatcher.handleUpEvent(MessageDispatcher.java:605)
        at org.jgroups.blocks.MessageDispatcher$ProtocolAdapter.up(MessageDispatcher.java:686)
        ... 25 more
I have already stumbled upon this issue in Wildfly 9: https://developer.jboss.org/thread/265607